src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+    - `uri`: a URI of the configuration storage's instance.
+    - `login`: a username used to connect to the instance.
+    - `password`: a password used for authentication.
+    - `params`: SSL parameters required for encrypted connections
+]])
+
+M['config.storage.endpoints.*'] = format_text([[
+    Element that represents a configuration storage endpoint with the
+    following fields:
+
+    - `uri`: a URI of the configuration storage's instance.
+    - `login`: a username used to connect to the instance.
+    - `password`: a password used for authentication.
+    - `params`: SSL parameters required for encrypted connections.
+]])
+
+M['config.storage.endpoints.*.login'] = format_text([[
+    A username used to connect to the instance.
+]])
+
+M['config.storage.endpoints.*.params'] = M['<uri>.params']
+
+M['config.storage.endpoints.*.params.ssl_ca_file'] =
+    M['<uri>.params.ssl_ca_file']
+
+M['config.storage.endpoints.*.params.ssl_cert_file'] =
+    M['<uri>.params.ssl_cert_file']
+
+M['config.storage.endpoints.*.params.ssl_ciphers'] =
+    M['<uri>.params.ssl_ciphers']
+
+M['config.storage.endpoints.*.params.ssl_key_file'] =
+    M['<uri>.params.ssl_key_file']
+
+M['config.storage.endpoints.*.params.ssl_password'] =
+    M['<uri>.params.ssl_password']
+
+M['config.storage.endpoints.*.params.ssl_password_file'] =
+    M['<uri>.params.ssl_password_file']
+
+M['config.storage.endpoints.*.params.transport'] =  M['<uri>.params.transport']
+
+M['config.storage.endpoints.*.password'] = 'A password used for authentication.'
+
+M['config.storage.endpoints.*.uri'] = format_text([[
+    A URI of the configuration storage's instance.
+]])
+
+M['config.storage.prefix'] = format_text([[
+    A key prefix used to search a configuration in a centralized configuration
+    storage. Tarantool searches keys by the following path: `<prefix>/config/*`.
+    Note that `<prefix>` should start with a slash (`/`).
+]])
+
+M['config.storage.reconnect_after'] = format_text([[
+    A number of seconds to wait before reconnecting to a configuration storage.
+]])
+
+M['config.storage.timeout'] = format_text([[
+    The interval (in seconds) to perform the status check of a configuration
+    storage.
+]])
+
+-- }}} config.storage configuration
+
+-- }}} config configuration
+
+-- {{{ console configuration
+
+M['console'] = format_text([[
+    Configure the administrative console. A client to the console is `tt`
+    connect.
+]])
+
+M['console.enabled'] = format_text([[
+    Whether to listen on the Unix socket provided in the console.socket option.
+
+    If the option is set to `false`, the administrative console is disabled.
+]])
+
+M['console.socket'] = format_text([[
+    The Unix socket for the administrative console.
+
+    Mind the following nuances:
+
+    1. Only a Unix domain socket is allowed. A TCP socket can't be configured
+    this way.
+
+    2.console.socket is a file path, without any `unix:` or `unix/:` prefixes.
```

Comment:
Missed whitespace after `2.`.

Reply:
Replaced with an unordered list for consistency with other descriptions.